go/teams/service_helper.go

```diff
@@ -709,6 +727,38 @@ func AcceptSeitan(ctx context.Context, g *libkb.GlobalContext, ikey SeitanIKey)
 	return err
 }
 
+func AcceptSeitanV2(ctx context.Context, g *libkb.GlobalContext, ikey SeitanIKeyV2) error {
+	me, err := libkb.LoadMe(libkb.NewLoadUserArgWithContext(ctx, g))
+	if err != nil {
+		return err
+	}
+
+	sikey, err := ikey.GenerateSIKey()
+	if err != nil {
+		return err
+	}
+
+	inviteID, err := sikey.GenerateTeamInviteID()
+	if err != nil {
+		return err
+	}
+
+	now := keybase1.ToTime(time.Now())
+	_, encoded, err := sikey.GenerateSignature(me.GetUID(), me.GetCurrentEldestSeqno(), inviteID, now)
+	if err != nil {
+		return err
+	}
+
+	g.Log.CDebugf(ctx, "seitan invite ID: %v", inviteID)
+
+	arg := apiArg(ctx, "team/seitan_v2")
+	arg.Args.Add("sig", libkb.S{Val: encoded})
+	arg.Args.Add("now", libkb.S{Val: strconv.FormatInt(int64(now), 10)})
```

Comment:
Why not `libkb.I{Val:now}`?

Reply:
fixed

Reply:
Why don't you make a `libkb.HTTPTime` then that handles this properly....

Reply:
(in libkb/http.go)

Reply:
sorry, probably my fault, I recall having code like this in seitan v1

Reply:
done